e2e/flows/vue.e2e.js

```diff
@@ -0,0 +1,80 @@
+import chai, { expect } from 'chai';
+import path from 'path';
+import Helper from '../e2e-helper';
+
+chai.use(require('chai-fs'));
+
+describe('support vue files', function () {
+  this.timeout(0);
+  const helper = new Helper();
+
+  after(() => {
+    helper.destroyEnv();
+  });
+
+  describe('tests scenarios ', () => {
+    before(() => {
+      helper.setNewLocalAndRemoteScopes();
+      helper.copyFixtureComponents('vue');
+    });
+    describe('add vue files', () => {
+      before(() => {
+        helper.addComponent(path.normalize('directives/*.js'));
+        helper.addComponent(path.normalize('styles/*'));
+        helper.addComponent(path.normalize('UiAutocomplete.vue'));
+        helper.runCmd('npm i fuzzysearch');
```

Comment:
you can use the helper.addNpmPackage it will be much faster. you don't need to really install it.